.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      next-version:
+        type: choice
+        options:
+          - bump-minor
+          - bump-patch
+
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    with:
+      mode: release
+
+  release-to-github-and-bump:
+    uses: gardener/cc-utils/.github/workflows/release.yaml@master
```

Comment:
Do you cut releases in `cc-utils` or should we pin the references to some tag/hash to ensure reproducability and controlled updates?

Reply:
Now that I think of it, we could even include the commit-pinning into the `release.yaml`-workflow so that this will be done automatically upon release, as part of the release-commit. WDYT?